java/lang/Double/Double.cpp

```diff
@@ -139,22 +139,19 @@ boolean Double::operator>(const Double &target) const {
 }
 
 boolean Double::operator>=(const Double &target) const {
-	if (compare(this->doubleValue(), target.doubleValue()) == 1
-	    || compare(this->doubleValue(), target.doubleValue()) == 0) {
-		
-		return true;
-	}
-	return false;
+    if (compare(this->doubleValue(), target.doubleValue()) == 1
+            || compare(this->doubleValue(), target.doubleValue()) == 0) {
+        return true;
+    }
+    return false;
 }
 
 boolean Double::operator<=(const Double &target) const {
-	
-	if (compare(this->doubleValue(), target.doubleValue()) == -1
-	    || compare(this->doubleValue(), target.doubleValue()) == 0) {
-		
-		return true;
-	}
-	return false;
+    if (compare(this->doubleValue(), target.doubleValue()) == -1
```

Comment:
Please help me shorten this and add `this->` for compare

Reply:
Edited. Thanks!